src/main/java/frc/robot/subsystems/intake/PassToScorer.java

```diff
@@ -20,7 +20,7 @@ public class PassToScorer extends Command {
   private IntakeSubsystem intakeSubsystem;
   private BooleanSupplier elevatorNotL1;
   private CoralScorer coralScorer;
-  private Supplier<CoralEnum> state;
+  public Supplier<CoralEnum> state;
```

Comment:
You can just delete this variable and its associated parameter since it is unused.